src/Redistribution.sol

```diff
@@ -162,14 +162,9 @@ contract Redistribution is AccessControl, Pausable {
 
     // Next two events to be removed after testing phase pending some other usefulness being found.
     /**
-     * @dev Emits the number of commits being processed by the claim phase.
+     * @dev Emits the number of commits and reveals being processed by the claim phase.
      */
-    event CountCommits(uint256 _count);
-
-    /**
-     * @dev Emits the number of reveals being processed by the claim phase.
-     */
-    event CountReveals(uint256 _count);
+    event CountCommitsReveals(uint256 _count, uint256 _count);
```

Comment:
we should avoid changing events unless we need to in order not to break compatibility

Reply:
makes sense, will put it as it was.